solidity/build/TestNonStandardERC20Token.bin

```diff
@@ -0,0 +1 @@
23575b600080fd5b3480156100a957600080fd5b506100b261024a565b6040805160208082528351818301528351919283929083019185019080838360005b838110156100ec5781810151838201526020016100d4565b50505050905090810190601f1680156101195780820380516001836020036101000a031916815260200191505b509250505060405180910390f35b34801561013357600080fd5b5061014b600160a060020a03600435166024356102d8565b005b34801561015957600080fd5b5061016261038a565b60408051918252519081900360200190f35b34801561018057600080fd5b5061014b600160a060020a0360043581169060243516604435610390565b3480156101aa57600080fd5b506101b36104c4565b6040805160ff9092168252519081900360200190f35b3480156101d557600080fd5b50610162600160a060020a03600435166104cd565b3480156101f657600080fd5b506100b26104df565b34801561020b57600080fd5b5061014b600160a060020a0360043516602435610539565b34801561022f57600080fd5b50610162600160a060020a03600435811690602435166105fb565b6000805460408051602060026001851615610100026000190190941693909304601f810184900484028201840190925281815292918301828280156102d05780601f106102a5576101008083540402835291602001916102d0565b820191906000526020600020905b8154815290600101906020018083116102b357829003601f168201915b505050505081565b81600160a060020a03811615156102ee57600080fd5b81158061031c5750336000908152600560209081526040808320600160a060020a0387168452909152902054155b151561032757600080fd5b336000818152600560209081526040808320600160a060020a03881680855290835292819020869055805186815290519293927f8c5be1e5ebec7d5bd14f71427d1e84f3dd0314c0f7b2291e5b200ac8c7c3b925929181900390910190a3505050565b60035481565b82600160a060020a03811615156103a657600080fd5b82600160a060020a03811615156103bc57600080fd5b600160a060020a03851660009081526005602090815260408083203384529091529020546103f0908463ffffffff61061816565b600160a060020a038616600081815260056020908152604080832033845282528083209490945591815260049091522054610431908463ffffffff61061816565b600160a060020a038087166000908152600460205260408082209390935590861681522054610466908463ffffffff61062d16565b600160a060020a0380861660008181526004602090815260409182902094909455805187815290519193928916927fddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef92918290030190a35050505050565b60025460ff1681565b60046020526000908152604090205481565b60018054604080516020600284861615610100026000190190941693909304601f810184900484028201840190925281815292918301828280156102d05780601f106102a5576101008083540402835291602001916102d0565b81600160a060020a038116151561054f57600080fd5b3360009081526004602052604090205461056f908363ffffffff61061816565b3360009081526004602052604080822092909255600160a060020a038516815220546105a1908363ffffffff61062d16565b600160a060020a0384166000818152600460209081526040918290209390935580518581529051919233927fddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef9281900390910190a3505050565b600560209081526000928352604080842090915290825290205481565b60008183101561062757600080fd5b50900390565b60008282018381101561063f57600080fd5b93925050505600a165627a7a72305820b8e5fe5b4fef6ab67df311f68a61ee2fb50edf616be00021318b318a2004ba4f0029
```

Comment:
@yudilevi : Should be deleted from the repository.

Reply:
removed